src/index.ts

```diff
@@ -13,16 +14,34 @@ app.use(async (ctx, next) => {
 });
```

Comment:
Do we still need this?

Reply:
Still needed by the router middleware for handling actual request.